submitter_config.py

```diff
@@ -127,7 +130,7 @@ def mapping(self, template=None):
                                   _list_inter.append({item: obj})
                       if _list_inter:
                          _for_dic[key_inter] = _list_inter
-                         _for_dic['dry_run'] = self.main_config['dry_run']
+                         #_for_dic['dry_run'] = self.main_config['dry_run']
```

Comment:
Deleting non-required comments:
#_for_dic['dry_run']  as dry_run is no longer in key_config